File: Ad-Hoc/2187/2187.c
#include <stdio.h>

int main(){
    int n, b50, b10, b5, b1, count = 1;
    scanf("%d", &n);
    while (n!=0){
        b50 = n/50;
        b10 = n%50/10;
        b5 = n%50%10/5;
        b1 = n%50%10%5;
        printf("Teste %d\n%d %d %d %d\n\n",count, b50, b10, b5, b1);
        count++;
        scanf("%d", &n);
    }
    return 0;
}

File: Ad-Hoc/2187/2187.py
V = int(input())
count = 1
while V>0 and V<10000:
    B50 = V//50
    B10 = (V%50)//10
    B5 = (V%50%10)//5
    B1 = V%50%10%5
    print(f"Teste {count}")
    print(f"{B50} {B10} {B5} {B1}")
    print()
    count+=1
    V = int(input())